docs/source/guide/cdr-1-intro.myst

```diff
@@ -0,0 +1,153 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use CDR?
+
+Here we show how to use Clifford data regression (CDR) by means of a simple example. CDR is a learning-based quantum error mitigation technique that relies on the training of a classical simulator with quantum circuits that resemble the one that is the aim of the quantum error mitigation, but which are easier to simulate {cite}`Czarnik_2021_Quantum`. More information about this technique can be found in the section [What is the theory behind CDR?](cdr-5-theory.myst).
```

Comment:
```suggestion
Clifford data regression (CDR) is a learning-based quantum error mitigation technique in which an error mitigation model is trained with quantum circuits that _resemble_ the circuit of interest, but which are easier to classically simulate {cite}`Czarnik_2021_Quantum, Lowe_2021_PRR`. More information about this technique can be found in the section [What is the theory behind CDR?](cdr-5-theory.myst).

Here we show how to use CDR by means of a simple example.
```
The initial sentence is a bit confusing since it says that we "train a classical simulator". Actually one trains a mitigation model "using" a simulator.